Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -35,38 +35,65 @@ private struct RemoteTabsPanelUX {
     static let EmptyStateInstructionsSyncTabsPasswordsString = NSLocalizedString("Sync your tabs, passwords and more.", comment: "Sync tabs and passwords empty state instructions.")
 
     static let EmptyStateInstructionsGetTabsBookmarksPasswordsString = NSLocalizedString("Get your open tabs, bookmarks, and passwords from your other devices.", comment: "A re-worded offer about Sync that emphasizes one-way data transfer, not syncing.")
+
+    static let HistoryTableViewHeaderChevronInset: CGFloat = 10
+    static let HistoryTableViewHeaderChevronSize: CGFloat = 20
+    static let HistoryTableViewHeaderChevronLineWidth: CGFloat = 3.0
 }
 
 private let RemoteClientIdentifier = "RemoteClient"
 private let RemoteTabIdentifier = "RemoteTab"
 
-class RemoteTabsPanel: UITableViewController, HomePanel {
+class RemoteTabsPanel: UIViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
+    private lazy var tableViewController = {
+        return UITableViewController()
+    }()
+    private lazy var historyBackButton: HistoryTableViewHeader = {
+        return HistoryTableViewHeader()
+    }()
     var profile: Profile!
 
     init() {
         super.init(nibName: nil, bundle: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(RemoteTabsPanel.notificationReceived(_:)), name: NotificationFirefoxAccountChanged, object: nil)
     }
 
-    required init!(coder aDecoder: NSCoder) {
+    required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        tableView.registerClass(TwoLineHeaderFooterView.self, forHeaderFooterViewReuseIdentifier: RemoteClientIdentifier)
-        tableView.registerClass(TwoLineTableViewCell.self, forCellReuseIdentifier: RemoteTabIdentifier)
+        tableViewController.tableView.registerClass(TwoLineHeaderFooterView.self, forHeaderFooterViewReuseIdentifier: RemoteClientIdentifier)
+        tableViewController.tableView.registerClass(TwoLineTableViewCell.self, forCellReuseIdentifier: RemoteTabIdentifier)
 
-        tableView.rowHeight = RemoteTabsPanelUX.RowHeight
-        tableView.separatorInset = UIEdgeInsetsZero
+        tableViewController.tableView.rowHeight = RemoteTabsPanelUX.RowHeight
+        tableViewController.tableView.separatorInset = UIEdgeInsetsZero
 
-        tableView.delegate = nil
-        tableView.dataSource = nil
+        tableViewController.tableView.delegate = nil
+        tableViewController.tableView.dataSource = nil
```

Comment:
Nit: Maybe just a personal thing but I don't really like reaching into an object to setup properties on one of it's properties. The parent shouldn't need to understand the internals of the tableViewController. A simple workaround could be to add a private extension inside this file on `UITableViewController` that adds a custom class method that contains this setup code. If you find you're making a lot of uses of the tableViewController you can also create a subclasses.